e2e/helpers/SauceLabsReportingHelper.js

```diff
@@ -9,21 +9,31 @@ function updateSauceLabsResult(result, sessionId) {
   return 'curl -X PUT -s -d \'{"passed": ' + result + '}\' -u ' + process.env.SAUCE_USERNAME + ':' + process.env.SAUCE_ACCESS_KEY + ' https://eu-central-1.saucelabs.com/rest/v1/' + process.env.SAUCE_USERNAME + '/jobs/' + sessionId;
 }
 
-module.exports = function() {
-
-  // Setting test success on SauceLabs
-  event.dispatcher.on(event.test.passed, () => {
-
-    const sessionId = container.helpers('WebDriver').browser.sessionId;
-    exec(updateSauceLabsResult('true', sessionId));
-
+module.exports = function () {
+  let failedTests = [];
```

Comment:
would it be easier to use as [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) instead?

Reply:
yep, waaaaay easier 🤦🏻‍♂️ 
will change

Reply:
done